open_instruct/finetune.py

```diff
@@ -451,28 +451,27 @@ def main(args: FlatArguments):
     else:
         raise ValueError("You need to have either 'prompt'&'completion' or 'messages' in your column names.")
 
+    train_dataset = raw_datasets["train"]
+
+    # debugging tool for fewer samples
+    if args.max_train_samples is not None:
+        max_train_samples = min(len(train_dataset), args.max_train_samples)
+        logger.info(f"Limiting training samples to {max_train_samples} from {len(train_dataset)}.")
+        train_dataset = train_dataset.select(range(max_train_samples))
+
     with accelerator.main_process_first():
-        lm_datasets = raw_datasets.map(
+        train_dataset = train_dataset.map(
```

Comment:
In `finetune.py` the other splits are never really used, so I just replacing with train_dataset